src/uct/api/uct_def.h

```diff
@@ -116,37 +116,19 @@ typedef void*                      uct_conn_request_h;
  * @ingroup UCT_RESOURCE
  * @brief Structure for scatter-gather I/O.
  *
- * Specifies a list of buffers which can be used within a single data transfer
+ * Specifies a buffer which can be used within a single data transfer
```

Comment:
`a list of buffers` means that we have multiple vectors packed in the IO vector

Reply:
I think this was about the case when count is greater than 1, since this is a description for the structure.

Reply:
I see